packages/components/src/context-tag/context-tag.tsx

```diff
@@ -288,6 +288,7 @@ const Base = styled(View, {
   display: 'inline-flex',
   flexDirection: 'row',
   alignItems: 'center',
+  cursor: 'default',
```

Comment:
til: unprefixed `user-select` is not supported in Safari, and using the prefix in the inspector didn't work for me either

<img width="818" alt="image" src="https://github.com/status-im/status-web/assets/13265126/1b01d85c-4c47-4afb-adc4-c34cf3377e84">